internal/pkg/cli/app_delete.go

```diff
@@ -188,6 +188,10 @@ func (o *deleteAppOpts) Ask() error {
 // It removes all the services from each environment, the environments, the pipeline S3 buckets,
 // the pipeline, the application, removes the variables from the config store, and deletes the local workspace.
 func (o *deleteAppOpts) Execute() error {
+	if err := o.deletePipelines(); err != nil {
```

Comment:
maybe a comment why it needs to go first? or just in the `Execute()` description

Reply:
Updated the function comment. I think it's pretty obvious. Deleting the pipelines in the middle (previous impl) should be more confusing because nothing depends on a pipeline.